### ISSUE TITLE
Define a custom DNS record for an SFTP endpoint

### ISSUE THREAD

**Issue by dimisjim** [hashicorp/terraform-provider-aws#19139]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

basically what is discussed in https://github.com/hashicorp/terraform-provider-aws/issues/6956
Not sure why it was closed, as this is still not supported by terraform

### New or Affected Resource(s)

* aws_transfer_server

**Comment by ewbankkit** [comment]
@dimisjim  Thanks for raising this issue.
We have an existing open issue for this: #18077. I'm going to close this one as a duplicate so that we can concentrate discussion in the linked issue.
Please add any additional comments there.
